spec/features/pom_caseload_spec.rb

```diff
@@ -49,6 +55,72 @@
     create(:allocation_version, nomis_offender_id: 'G4706UP', primary_pom_nomis_id: '485637', nomis_booking_id: '1180800')
     create(:allocation_version, nomis_offender_id: 'G9344UG', primary_pom_nomis_id: '485637', nomis_booking_id: '841994')
     create(:allocation_version, nomis_offender_id: 'G4273GI', primary_pom_nomis_id: '485637', nomis_booking_id: '1153753')
+
+    stub_request(:get, elite2listapi).
```

Comment:
I think we need to start removing the duplication in these tests real soon now [tm]